.github/workflows/scheduled.yml

```
@@ -260,6 +264,7 @@ jobs:
         with:
           flavor: integration
       - name: Start the dev infra stack (compose + app role)
+        if: steps.funded.outputs.configured == 'yes'
```

actionlint (1.7.12)

[error] property "funded" is not defined in object type {}

(expression)